print_sql_create_command.py

```diff
@@ -355,7 +383,7 @@ def main():
 
     print(db.schema_create_command)
 
-    for table in tables:
+    for table in sorted(tables, key=number_of_foreign_keys):
```

Comment:
quick and dirty way of having tables with FKs references tables that exist in the create command before. something better here would be nice